appserver/extras/payara-micro/payara-micro-distribution/pom.xml

```diff
@@ -518,6 +518,18 @@
                     <artifactId>microprofile-metrics</artifactId>
                     <version>${project.version}</version>
                 </dependency>
+                <!-- JWT -->
```

Comment:
Just a thought, could we change this so that we just grab the the microprofile-package?